common/account_utils/src/validator_definitions.rs

```diff
@@ -361,6 +396,30 @@ mod tests {
     use super::*;
     use std::str::FromStr;
 
+    #[test]
+    fn validator_definitions_file_locks() {
+        let validators_dir: PathBuf = ".".into();
```

Comment:
When we need an arbitrary directory we usually use a `tempfile::tempdir` to prevent conflicts between tests. The current version of `tempfile` that we're using seems to be 3.1.0

Reply:
@michaelsproul Is it worth mandating the current working directory (e.g., via [`TempDir::new_in`](https://docs.rs/tempfile/3.2.0/tempfile/struct.TempDir.html#method.new_in) or just the usual route of [`TempDir::new`](https://docs.rs/tempfile/3.2.0/tempfile/struct.TempDir.html#method.new)?

Reply:
TempDir::new or [`tempdir`](https://docs.rs/tempfile/3.2.0/tempfile/fn.tempdir.html) are what we usually use (it's nice to avoid test files in source directories).